docs/guides/building_on_windows.md

```diff
@@ -0,0 +1,202 @@
+
+### Installing vcpkg
+
+vcpkg is an open-source cross-platform package manager by Microsoft. It heavily relies on git and CMake to function. On Windows, it is the most convenient way to get dependencies for MSVC.
+
+Start off by opening a terminal and going to the location where you wish to install vcpkg. It should preferably a short path such as `C:\`, `D:\Dev`, or similar due to path limits, for the rest of this section, it will be assumed that vcpkg will be installed in `C:\`. Run the following commands to setup vcpkg:
+
+```
+cd C:
+git clone https://github.com/Microsoft/vcpkg.git
+cd vcpkg
+./bootstrap-vcpkg.bat -disableMetrics
+```
+
+You then want to add the directory you chose to your PATH, in this case, you would add `C:\vcpkg\`. After this, if you close and re-open your terminal, you should be able to use the `vcpkg` command from anywhere.
+
+If you plan on using Visual Studio, open a terminal **as Administrator**, and run the following command:
+
+```
+vcpkg integrate install
+```
+
+**IMPORTANT: vcpkg does not update itself automatically.** To get newer versions of libraries, you will need to regularly come back and run `git pull`. Similarly, to update the tool, run `./bootstrap-vcpkg.bat -disableMetrics` again after pulling the changes.
+
+### Building with Visual Studio
+
+Open Visual Studio, and select the "Clone a repository" option. The repository location is `https://github.com/kraflab/dsda-doom.git`, and the path you provide is where the files will be copied to, for example `C:/git/dsda-doom/`. If you already have cloned the repository beforehand, select "Open" and "Open Folder", you should **not** use the "CMake" option as this repo uses an uncommon layout.
+
+To generate the CMake cache, select "Project" and then "Configure Cache". During configuration, the vcpkg toolchain will be automatically provided to CMake and it will compile all the dependencies. Once it finishes, you should be able to select `dsda-doom.exe` as a startup item and run it.
+
+By default, the CMake integration only provides an `x64-Debug` configuration. To add a release configuration, head back to "Project" and select "CMake settings for dsda-doom".
+
+On the left, you should see a list of all the configurations you have, click the green + at the top and search for `x64-Release`. After adding it, you should now be able to switch configuration at the top. This menu also lets you customise the CMake cache and set defaults.
+
+To install a build, select "Build" and then "Install dsda-doom". The default directories for build and install are, respectively, `out/build/<config>` and `out/install<config>`.
+
+### Building from the terminal
+
+Make a clone of the repository:
+
+```
+git clone https://github.com/kraflab/dsda-doom.git
+```
+
+Run the CMake configuration:
+
+```
+cd dsda-doom
+cmake -Sprboom2 -Bbuild -DCMAKE_TOOLCHAIN_FILE="C:\vcpkg\scripts\buildsystems\vcpkg.cmake" -DCMAKE_BUILD_TYPE=Release
```

Comment:
Why do you recommend the `Release` configuration? Developers probably want `Debug`. Also you are using MS Visual Studio generator here, it will configure both `Debug` and `Release` anyway.